### PR TITLE
Remove help page, replace help link

### PR DESCRIPTION
**Pull request by tonoyansergey-devt**

### CODE REVIEW

**Comment by tonoyansergey-devt** [comment]
- Help page removed
- Created a redirection from /help page to https://www.travis-ci.com/resources/ page
- Two configurable variables to handle the redirection (example on staging)
| Variable | Value | Description |
|--------|--------|--------|
| APP_ENDPOINT | app-staging-dev1.travis-ci.com | The address of the host from where the redirection will happen. 
e.g. https://**app-staging-dev1.travis-ci.com**/help |
| TRAVIS_HELP_REDIRECT_URL | https://www.travis-ci.com/resources/ | The target url of redirection |

File: app/templates/components/billing-manual.hbs [hunk 1]
```diff
@@ -27,9 +27,9 @@
           </div>
           <span class="py-1 leading-loose" data-test-manual-subscription-banner='true'>
             This manual subscription is paid to Travis CI by bank transfer. If you have any questions or would like to update your plan,
-            <LinkTo @route='help' @query={{hash anchor='form' billing=true}}>
+            <ExternalLinkTo @href={{config-get 'urls.resources'}} @title="Travis CI Resources">
```
Comment (gbarc-dt) [inline app/templates/components/billing-manual.hbs:30]: where's urls.resources defined?

File: app/templates/components/billing/payment-details-tab.hbs [hunk 2]
```diff
@@ -249,9 +249,9 @@
         <p class="invoice-issue-banner invoice-issue-banner-no-invoices" data-test-help-text='true'>
           <SvgImage @name='icon-help' @class='icon-help' />
           Having trouble with your invoices?
-          <LinkTo @route='help' @query={{hash anchor='form' billing=true}}>
+          <ExternalLinkTo @href={{config-get 'urls.resources'}} @title="Travis CI Resources">
```
Comment (gbarc-dt) [inline app/templates/components/billing/payment-details-tab.hbs:252]: this is what's rendered here (no urls.resources defined):
<a title="Travis CI Resources" rel="noopener noreferrer" target="_blank" id="ember644" class="ember-view">  
        We’re happy to help
      
</a>

File: app/templates/components/billing-manual.hbs (hunk 1 above)
Comment (tonoyansergey-devt) [reply app/templates/components/billing-manual.hbs:30]: Added missing part - https://github.com/travis-ci/travis-web/pull/2922/commits/09d17216c9f7b78cc5d50738d6b9448639b8f8b5

File: app/templates/components/billing/payment-details-tab.hbs (hunk 2 above)
Comment (tonoyansergey-devt) [reply app/templates/components/billing/payment-details-tab.hbs:252]: Added missing part - https://github.com/travis-ci/travis-web/pull/2922/commits/09d17216c9f7b78cc5d50738d6b9448639b8f8b5

Comment (tonoyansergey-devt) [reply app/templates/components/billing/payment-details-tab.hbs:252]: image

**Pull request merged**